src/features/HelpAndKnowledge/Articles/index.tsx

```diff
@@ -55,7 +53,7 @@ const helpAndKnowledgeArticles: ArticleProps[] = [
     key: HelpArticle.SIX_PAGER_MEETING,
     route: '/how-to-have-a-successful-6-pager-meeting',
```

Comment:
Route is still has the old title

![image](https://github.com/user-attachments/assets/3ec5fede-deda-4fa6-a9b5-59296099e64e)